### PR TITLE
created sin and cosine functions

### PR DESCRIPTION
**Pull request by pchapl1**
added sine and cosine functions

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -22,19 +23,24 @@ def sqrt(n):
...
-	args = parser.parse_args()
-
-	parser.add_argument('--add', nargs=2, type=int, help='adds two numbers')
+	parser.add_argument('--sub', nargs=2, type=int, help='subtracts two numbers')
 	parser.add_argument('--divide', nargs=2, type=int, help='divides two numbers')
-	parser.add_argument('--remainder',nargs=2, type=int, help='returns the remainder of two numbers')
+	parser.add_argument('--remainder', nargs=2, type=int, help='returns the remainder of two numbers')
 	parser.add_argument('--sqrt', nargs =1, type=int, help='square roots a number')
 	parser.add_argument('--multiply', nargs='+', type=int, help='multiply more then one numbers')
+	parser.add_argument('--sine', nargs='+', type=int, help='returns the sine of a number')
```
Comment (dd-code-immersives) [inline calculator.py:42]: i would restrict the input of sine and cosine to one argument instead of 1 or more

Comment (dd-code-immersives) [reply calculator.py:42]: that goes for sine and cosine

**Review by dd-code-immersives — CHANGES_REQUESTED** [review]

**Review by dd-code-immersives — APPROVED** [review]

File: calculator.py (hunk 1 above)
Comment (pchapl1) [reply calculator.py]: restricted input to one arg for both sine and cosine

**Review by TreySchneider95 — APPROVED** [review]

**Comment by TreySchneider95** [comment]
nice

**Pull request merged**